repo/packages/K/kafka/0/marathon.json

```diff
@@ -0,0 +1,13 @@
+{
+  "id": "kafka",
+  "cpus": 0.5,
+  "mem": 256,
+  "instances": 1,
+  "cmd": "java -jar kafka-mesos*.jar scheduler --master=zk://master.mesos:2181/mesos --zk=master.mesos:2181 --api=http://localhost:$PORT0",
```

Comment:
I've tried using http://master.mesos:$PORT0/ but with no luch. 

For some reason http port of master is not accessible from slave while is it accessible from the outside, using domain name like: http://ec2-54-149-5-171.us-west-2.compute.amazonaws.com:10000 (10000 - assigned port).

What could be causing this?